app/models/split.rb

```diff
@@ -19,7 +19,9 @@ class Split < ActiveRecord::Base
 
   before_validation :cast_registry
 
-  scope :active, -> { where(finished_at: nil) }
+  scope :active, ->(as_of: nil) do
+    as_of ? where('splits.finished_at is null or splits.finished_at > ?', as_of) : where(finished_at: nil)
+  end
```

Comment:
Realized I already had the start of this scope, so I refactored it to take an optional argument, and use it to drive out the composable scope refactor you suggested on #98, @smudge.